src/test-utils/override-ipc-bridge.ts

```diff
@@ -101,4 +103,49 @@ export const overrideIpcBridge = ({
         handles.forEach((handle) => handle(...data));
       },
   );
+
+  const mainIpcFakeHandles = new Map<
+    string,
+    ((...args: any[]) => void)[]
+  >();
+
+  rendererDi.override(
+    enlistChannelListenerInjectableInRenderer,
+    () => (channel, handler) => {
+      const existingHandles = rendererIpcFakeHandles.get(channel.id) || [];
+
+      rendererIpcFakeHandles.set(channel.id, [...existingHandles, handler]);
+
+      return () => {
+
+      };
```

Comment:
?

Reply:
Disposer which is not that important in unit tests for know.